packages/client/src/components/ratings/FutureProofRatingView/FutureProofRatingView.tsx

```diff
@@ -1,15 +1,25 @@
 import CertificatesList from '@components/ratings/CertificateList'
 import CircularRatingIndicator from '@components/ratings/CircularRatingIndicator'
 import RatingBreakdownItems from '@components/ratings/RatingBreakdownItems'
```

Comment:
Do we need to maintain a breakdown and an Alt if we're just using one?

Reply:
The original component was written in a way that expected an array of metrics to be returned from the database. As this is not the way the database is structured, I created an Alt component. However, I decided to leave the original file there in case there database is changed in the future.

Reply:
Ok will remove.